cvat/apps/lambda_manager/views.py

```diff
@@ -234,45 +253,106 @@ def mandatory_arg(name: str) -> Any:
+                            'name': task_label.name,
+                            'attributes': attributes_default_mapping,
+                        }
+
+                        if model_label['type'] == 'skeleton' and task_label.type == 'skeleton':
+                            mapping_by_default[model_label['name']]['sublabels'] = _make_default_mapping(
+                                model_label['sublabels'],
+                                task_label.sublabels.all(),
+                            )
+
+            return mapping_by_default
+
+        def _update_mapping(_mapping, _model_labels, _db_labels):
+            copy = deepcopy(_mapping)
+            for model_label_name in copy:
+                mapping_item = copy[model_label_name]
+                md_label = next(filter(lambda x: x['name'] == model_label_name, _model_labels))
+                db_label = next(filter(lambda x: x.name == mapping_item['name'], _db_labels))
+                mapping_item['md_label'] = md_label
+                mapping_item['db_label'] = db_label
+                if md_label['type'] == 'skeleton' and db_label.type == 'skeleton':
+                    mapping_item['sublabels'] = _update_mapping(
+                        mapping_item['sublabels'],
+                        md_label['sublabels'],
+                        db_label.sublabels.all()
+                    )
+            return copy
+
+        def _validate_labels_mapping(_mapping, _model_labels, _db_labels):
+            def _validate_attributes_mapping(attributes_mapping, model_attributes, db_attributes):
+                db_attr_names = [attr.name for attr in db_attributes]
+                model_attr_names = [attr['name'] for attr in model_attributes]
+                for model_attr in attributes_mapping:
+                    task_attr = attributes_mapping[model_attr]
+                    if model_attr not in model_attr_names:
+                        raise ValidationError(f'Invalid mapping. Unknown model attribute "{model_attr}"')
+                    if task_attr not in db_attr_names:
+                        raise ValidationError(f'Invalid mapping. Unknown db attribute "{task_attr}"')
+
+            db_label_names = [label.name for label in _db_labels]
+            model_label_names = [label['name'] for label in _model_labels]
+            for model_label_name in _mapping:
+                mapping_item = _mapping[model_label_name]
+                db_label_name = mapping_item['name']
+                if model_label_name not in model_label_names:
+                    raise ValidationError(f'Invalid mapping. Unknown model label "{model_label_name}"')
+                if db_label_name not in db_label_names:
+                    raise ValidationError(f'Invalid mapping. Unknown db label "{db_label_name}"')
+
+                md_label = next(filter(lambda x: x['name'] == model_label_name, _model_labels))
+                db_label = next(filter(lambda x: x.name == mapping_item['name'], _db_labels))
```

Comment:
```suggestion
                try:
                    md_label = next(x for x in _model_labels if x['name'] == model_label_name)
                except StopIteration:
                    raise ValidationError(f'Invalid mapping. Unknown model label "{model_label_name}"')
                
                try:
                    db_label = next(x for x in _db_labels if x.name == db_label_name)
                except StopIteration:
                    raise ValidationError(f'Invalid mapping. Unknown db label "{db_label_name}"')
```

`db_label_names` and `model_label_names` can then be deleted.